config.h

```diff
@@ -0,0 +1,8 @@
+#ifndef CONFIG_H_

```

Comment:
Config is a very generic name. This file contains execution environment related definitions. Maybe some time later, some production code related configuration might get added which is not intended now!

Hence naming of the file should be specific to indicate the content.

Reply:
Updated review comments